layouts/default.vue

```diff
@@ -1,3 +1,34 @@
+<script setup lang="ts">
+type Theme = 'light' | 'dark'
+
+interface Message {
+  type: 'theme'
+  theme: Theme
+}
+
+function setTheme(theme: Theme) {
```

Comment:
Can we do it using [`useDark()`](https://vueuse.org/core/useDark/#basic-usage) which hides the implementation details?

Reply:
TIL, changed